python/python/res/res_util/__init__.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright (C) 2017  Statoil ASA, Norway.
```

Comment:
Since the python package provides namespacing - we could do `res.util` - I would suggest that. That is what is done in the libecl library.